browse/controllers/abs_page/__init__.py

```diff
@@ -129,6 +130,16 @@ def get_abs_page(arxiv_id: str) -> Response:
             abs_meta, download_format_pref, add_sciencewise_ping
         )
 
+        response_data["withdrawn_versions"] = []
```

Comment:
You should be able to detect if withdrawn with 

    ver_withdrawn = 'I' in abs_meta.version_history[i].source_type.code 

but what you have is fine.